cmd/loader/main.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"math/rand"
+	"net"
+	"os"
+	"strings"
+	"sync/atomic"
+	"time"
+)
+
+func main() {
+	opts := parseArgs(os.Args[1:])
+
+	pendingWorkers := make(chan struct{}, opts.Workers)
+	metricGenerators := make([]*metricGenerator, 0, opts.Workers)
+	for i := uint(0); i < opts.Workers; i++ {
+		generator := &metricGenerator{
+			rnd: rand.New(rand.NewSource(rand.Int63())),
```

Comment:
:fry: What? 

Would this be better to seed with `time.Now.UnixNano()`?

Reply:
`time.Now()` makes no guarantee of advancing when called rapidly in succession.  In all fairness, `rand.Int63()` makes no guarantee of not returning the same number twice in a row either.